src/isomorphic/children/ReactChildren.js

```diff
@@ -146,7 +146,7 @@ function countChildren(children, context) {
 }
```

Comment:
@spicyj mapChildren (above) returns `ReactFragment.create`. We should probably optimize it to just create an array immediately. This is probably a perf regression.